infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -399,18 +400,17 @@ public void handleHeartbeatRequest(CorfuMsg msg, ChannelHandlerContext ctx, ISer
     public synchronized CorfuRuntime getCorfuRuntime() {
 
         if (corfuRuntime == null) {
-            corfuRuntime = new CorfuRuntime();
-            if ((Boolean) opts.get("--enable-tls")) {
-                corfuRuntime.enableTls((String) opts.get("--keystore"),
-                        (String) opts.get("--keystore-password-file"),
-                        (String) opts.get("--truststore"),
-                        (String) opts.get("--truststore-password-file"));
-                if ((Boolean) opts.get("--enable-sasl-plain-text-auth")) {
-                    corfuRuntime.enableSaslPlainText(
-                            (String) opts.get("--sasl-plain-text-username-file"),
-                            (String) opts.get("--sasl-plain-text-password-file"));
-                }
-            }
+            CorfuRuntimeParameters params = CorfuRuntimeParameters.builder()
```

Comment:
I'd put this in serverContext, in case anything else happens to need a runtime. Maybe call it getDefaultRuntimeParameters or something?

Reply:
Done.